install-dependencies.sh

```diff
@@ -1,7 +1,37 @@
 #!/usr/bin/env sh
+PAZUZU_DIR=$GOPATH/src/github.com/zalando-incubator/pazuzu
+SWAGGER_DIR=$PAZUZU_DIR/swagger
+SWAGGER_FILE=$SWAGGER_DIR/swagger.yaml
+PAZUZU_REGISTRY_DIR=$PAZUZU_DIR/../pazuzu-registry/
+PAZUZU_REGISTRY_SWAGGER_LOCAL=$PAZUZU_REGISTRY_DIR/src/main/resources/api/swagger.yaml
+PAZUZU_REGISTRY_SWAGGER_UPSTREAM="https://raw.githubusercontent.com/zalando-incubator/pazuzu-registry/master/src/main/resources/api/swagger.yaml"
+
 go get -u github.com/kardianos/govendor
 go get -u github.com/go-swagger/go-swagger/cmd/swagger
-cd $GOPATH/src/github.com/zalando-incubator/pazuzu/
+
+cd $PAZUZU_DIR
 govendor sync
-cd $GOPATH/src/github.com/zalando-incubator/pazuzu/swagger
+
+get_swagger_api_def() {
```

Comment:
Just to reduce mental resources that required to read this script - it's not that cool that function definition is inside block of actual code. At least we can move it too at the beginning or end of the script.